core-services/prow/02_config/openshift-service-mesh/sail-operator/_prowconfig.yaml

```diff
@@ -0,0 +1,33 @@
+branch-protection:
+  orgs:
+    openshift-service-mesh:
+      repos:
+        sail-operator:
+          branches:
+            pre-main:
+              protect: true
+              required_pull_request_reviews:
+                required_approving_review_count: 1
+              unmanaged: false
+tide:
+  merge_method:
+    openshift-service-mesh/sail-operator: squash
+  queries:
+  - author: maistra-bot
```

Comment:
This can be renamed to another name that not include maistra? I mean, something like sail-bot for example

Reply:
Not renamed. We'd need to create another account for this. Not sure if it's high priority. We can do this in a follow up if necessary.